index.js

```diff
@@ -631,6 +633,21 @@ const publicApiProxy = new Proxy(publicApi, {
                     process.exit(1); // eslint-disable-line
                 }
             };
+        } else if (typeof target[prop] === 'undefined') {
+            // Find the property with the closest Levenshtein distance
+            let similarProperty;
+            let minDistance = Number.MAX_VALUE;
+            for (const apiProperty in target) {
+                const distance = levenshtein.get(apiProperty, prop);
+                if (distance <= minDistance) {
+                    similarProperty = apiProperty;
+                    minDistance = distance;
+                }
+            }
```

Comment:
If you type something totally crazy that is completely not close to anything, it will suggest *something*, right? In Symfony's core, we use "3" as the maximum distance (if it's more than 3, we don't recommend anything): https://github.com/symfony/symfony/blob/1bb2bc322bc1c2d0e6e70c2a0fa9fc0be90757fd/src/Symfony/Component/ExpressionLanguage/SyntaxError.php#L34.

It probably makes sense to do the same thing here

Reply:
![2017-09-13_23-01-01](https://user-images.githubusercontent.com/850046/30400658-d9a7dfa4-98d7-11e7-909a-bf2fefe8730f.png)

![2017-09-13_23-02-08](https://user-images.githubusercontent.com/850046/30400665-de95d5d4-98d7-11e7-9c08-ea4c753c43fc.png)

Reply:
to be exact, we don't use 3 most of the time. We use `strlen(input) / 3` (we have a few cases using 3)

Reply:
Ah, thanks! I've just changed that at sha: 2e216a465fdd42071e4d781b5b84ad47e5442267... and indeed, it is a bit more "forgiving" in appropriate situations.